src/art/anshuman3/script.js

```diff
@@ -0,0 +1,27 @@
+const charcter = document.getElementById('character');
+const charcontainer = charcter.getContext('2d');
+const charwid = charcter.width = 623;
+const charhit = charcter.height = 572;
+const charimg = new Image();
+charimg.src = "sprite.png";
+const spritewid = 623 / 12;
+const sprithit = 572 / 8;
+let spriteX = 0;
+let spriteY = 0;
+let speedvar = 0;
+// c=0;
```

Comment:
Please remove unuseful codes or line here (line 12). Or you can add the effect of line 12 if uncommented. Thank you